images/proxy/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+ARG CILIUM_RUNTIME_IMAGE=quay.io/cilium/cilium-runtime:b32efb0e682e74311b628df328dfd768108b92c2@sha256:0873607c0fc991c884bc7d39ea774c8ebd7df2be35f4b69b6d16b33ce871427b
```

Comment:
good point. maybe in combination with "reducing" the base runtime image of envoy. then we could just build on top of this image when building the quay.io/cilium/proxy image in the cilium/cilium repo (which still makes sense to keep the versioning aligned with cilium/cilium) - instead of just copying files.